doc/architectural/cbmc-architecture.md

```diff
@@ -59,26 +59,24 @@ transformation tools (see \ref section-other-tools).
 
 To be documented.
 
-## Instrumentation (for test gen & CBMC & ...): Goto functions -> goto functions ##
+## Instrumentation: goto functions → goto functions ##
 
 To be documented.
 
-## Goto functions -> BMC -> Counterexample (trace) ##
+## Goto functions → BMC → Counterexample (trace) ##
```

Comment:
Are these arrow symbols safe across encodings/native locales? e.g. what happens on a JIS system? do the get correctly translated into HTML entities? (sorry if this is a can of worms)

Reply:
No idea :). It works in the HTML. I'm happy to change them all to "->" - I just want them to be consistent.

Reply:
A quick web search suggests the way to (portably) do this in Markdown is `&rarr;` - could you please give it a try?

Reply:
Done